src/dbos.ts

```diff
@@ -2058,34 +2062,6 @@ export class DBOS {
   ): R {
     return configureInstance(cls, name, ...args);
   }
-
-  // Function registration - for internal use
-  static registerAndWrapDBOSFunction<This, Args extends unknown[], Return>(
```

Comment:
`registerAndWrapDBOSFunction`, `executeWorkflowById` and `recoverPendingWorkflows` are all comments as being for internal and/or testing use only. `registerAndWrapDBOSFunction` is never used and the other two were simple enough to convert to test helpers.

This is technically a breaking change, but we should never have exposed or documented these anyway